.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@coder/coder-docs
```

Comment:
does this ping coder-docs for all changes in this repo?

Reply:
Yea, which might not be the best granularity (we may want to separate owners for CI or other aspects). Very open to ideas, I opened this to start that convo ✨

Reply:
I think it should be fine? It's just us at this moment and I can handle the (infrequent CI) stuff. It's not the busiest of repos